examples/webgl_materials_physical_clearcoat.html

```diff
@@ -1,245 +1,239 @@
-						diffuse.encoding = THREE.sRGBEncoding;
-						diffuse.wrapS = THREE.RepeatWrapping;
-						diffuse.wrapT = THREE.RepeatWrapping;
-						diffuse.repeat.x = 10;
-						diffuse.repeat.y = 10;
-
-						var normalMap = textureLoader.load( "textures/carbon/Carbon_Normal.png" );
-						normalMap.wrapS = THREE.RepeatWrapping;
-						normalMap.wrapT = THREE.RepeatWrapping;
-
-						var normalMap2 = textureLoader.load( "textures/water/Water_1_M_Normal.jpg" );
-
-						var normalMap3 = textureLoader.load( "textures/flakes.png" );
-						normalMap3.wrapS = THREE.RepeatWrapping;
-						normalMap3.wrapT = THREE.RepeatWrapping;
-						normalMap3.repeat.x = 10;
-						normalMap3.repeat.y = 10;
-						normalMap3.anisotropy = 16;
-
-						var normalMap4 = textureLoader.load( "textures/golfball.jpg" );
-
-						var clearcoatNormaMap = textureLoader.load( "textures/pbr/Scratched_gold/Scratched_gold_01_1K_Normal.png" );
-
-						// car paint
-
-						var material = new THREE.MeshPhysicalMaterial( {
-							clearcoat: 1.0,
-							clearcoatRoughness: 0.1,
-							metalness: 0.9,
-							roughness: 0.5,
-							color: 0x0000ff,
-							envMap: hdrCubeRenderTarget.texture,
-							normalMap: normalMap3,
-							normalScale: new THREE.Vector2( 0.1, 0.1 )
-						} );
-
-						var mesh = new THREE.Mesh( geometry, material );
-						mesh.position.x = - 100;
-						mesh.position.y = 100;
-						group.add( mesh );
-
-						// fibers
-
-						var material = new THREE.MeshPhysicalMaterial( {
-							clearcoat: 1.0,
-							clearcoatRoughness: 0.1,
-							envMap: hdrCubeRenderTarget.texture,
-							map: diffuse,
-							normalMap: normalMap
-						} );
-						var mesh = new THREE.Mesh( geometry, material );
-						mesh.position.x = 100;
-						mesh.position.y = 100;
-						group.add( mesh );
-
-						// golf
-
-						var material = new THREE.MeshPhysicalMaterial( {
-							metalness: 0.0,
-							roughness: 0.1,
-							clearcoat: 1.0,
-							normalMap: normalMap4,
-							envMap: hdrCubeRenderTarget.texture,
-							clearcoatNormalMap: clearcoatNormaMap,
-							clearcoatNormalScale: new THREE.Vector2( 2.0, 2.0 )
-						} );
-						var mesh = new THREE.Mesh( geometry, material );
-						mesh.position.x = - 100;
-						mesh.position.y = - 100;
-						group.add( mesh );
-
-						// clearcoat + normalmap
-
-						var material = new THREE.MeshPhysicalMaterial( {
-							clearcoat: 1.0,
-							metalness: 1.0,
-							color: 0xff0000,
-							envMap: hdrCubeRenderTarget.texture,
-							normalMap: normalMap2,
-							normalScale: new THREE.Vector2( 0.15, 0.15 ),
-							clearcoatNormalMap: clearcoatNormaMap,
-							clearcoatNormalScale: new THREE.Vector2( 2.0, 2.0 )
-						} );
-						var mesh = new THREE.Mesh( geometry, material );
-						mesh.position.x = 100;
-						mesh.position.y = - 100;
-						group.add( mesh );
-
-						//
-
-						scene.background = hdrCubeRenderTarget.texture;
-
-					}
-
+					.setPath( './textures/cube/pisaHDR/' )
```

Comment:
Do we have to go back to Pisa? It barely qualifies as HDR...

Reply:
I like how these spheres look in that HDR 😅